packages/docs/.vuepress/config.js

```diff
@@ -64,6 +57,11 @@ module.exports = {
           { title: 'Top Bar', link: '/components/top-bar', isVisible: false },
         ],
       },
+      {
+        children: [
+          { title: 'Changelog', link: 'https://github.com/okta/odyssey/blob/master/packages/odyssey/CHANGELOG.md', isVisible: true},
```

Comment:
Why redirect visitors to GitHub instead of the local copy in the docs site?

Reply:
Temporary. We didn't want to add VuePress markdown slots to the CHANGELOG markdown file. This is a middle ground while we figure out how we want to incorporate changelog into the next docs site